hum.php

```diff
@@ -317,7 +317,7 @@ public function get_shortlink( $link, $id, $context, $allow_slugs ) {
 			} elseif ( is_singular() ) {
 				$post_id = get_queried_object_id();
 			}
-		} elseif ( 'post' === $context && $id > 0 ) {
+		} elseif ( 'post' === $context && (int) $id > 0 ) {
```

Comment:
Technically this should not be required since $id will be delivered as int with default 0, but it won't hurt.